Dockerfile.windows.x86_64

```diff
@@ -0,0 +1,32 @@
+# escape=`
+
+ARG WIN_VER="ltsc2016"
```

Comment:
Is there a reason why this doesn't use `ltsc2019`? Maybe this PR was submitted before GA...

This [blog post](https://stefanscherer.github.io/docker-on-windows-server-2019/) suggests smaller base images. I suspect it might also be easier to run them as native containers (rather than Hyper-V containers), but currently don't have a setup to check.

Reply:
Indeed, the 2019 image was not publicly available at the time. Anyway, that's just a default. Any version can be requested when building the image, using "--build-arg".

Worth mentioning that the container Windows version cannot be newer than the host one, not even when using Hyper-V isolation.

Reply:
Started having issues with WS2016 after some Windows updates so I've switched the default as well to ltsc2019.